test/verify/check-storage-mdraid

```diff
@@ -81,10 +76,11 @@ class TestStorage(StorageCase):
                 except Error as ex:
                     if not ex.msg.startswith('timeout'):
                         raise
-                    self.browser.wait_present(".modal-dialog")
-                    self.browser.wait_in_text(".modal-dialog", "Error stopping RAID array /dev/md127")
-                    self.browser.click(".modal-dialog div.modal-footer button.btn-default")
-                    self.browser.wait_not_present(".modal-dialog")
+                    print("Stopping failed, retrying...")
+                    if self.browser.is_present("#dialog"):
+                        self.dialog_wait_error("Error stopping RAID array /dev/md127")
```

Comment:
Definition of `dialog_wait_error` needs two arguments (`def dialog_wait_error(self, field, val)`)

Reply:
Good catch, thanks!